maestrowf/datastructures/yamlspecification.py

```diff
@@ -95,22 +95,33 @@ def load_specification(cls, path):
         try:
             # Load the YAML spec from the file.
             with open(path, 'r') as data:
-                try:
-                    spec = yaml.load(data, yaml.FullLoader)
-                except AttributeError:
-                    logger.warning(
-                        "*** PyYAML is using an unsafe version with a known "
-                        "load vulnerability. Please upgrade your installation "
-                        "to a more recent version! ***")
-                    spec = yaml.load(data)
-
+                specification = cls.load_specification_from_stream(data)
         except Exception as e:
             logger.exception(e.args)
             raise
 
+        return specification
+
+    @classmethod
+    def load_specification_from_stream(cls, stream):
+        """
+        Load a study specification.
+
+        :param path: Path to a study specification.
```

Comment:
Update this docstring from `path` to `stream`.

Reply:
Also update the `returns` line in the same way.

Reply:
Good catch, corrected.